src/modules/EventList/index.tsx

```diff
@@ -433,6 +438,67 @@ export const EventList = (_props: EventListProps): ReactElement => {
     useAutoTrack('Click', 'Event Filter By', { filterBy: state.filterByTypes }, [state.filterByTypes]);
     useAutoTrack('Submit', 'Event Search', { searchTerm: state.searchTerm }, [state.searchTerm], !state.searchTerm);
 
+    // Check AlertList against current time
+    // and fire events
+    const { alertList, removeDateKey } = useAlertList();
+    useEffect(() => {
```

Comment:
Mind moving this out to it's own function/hook outside of the component and then just calling it here like `useEventAlerts()` or something? Not a big deal, but it's long/complicated enough to split out I think.

Reply:
Also wondering if any of this logic can be wrapped up into the `useAlertList()` hook. Maybe an optional arg to setup the interval and do the work in there? What do you think?

Reply:
👍